core/jazz_usermanagement/index.js

```diff
@@ -179,7 +180,7 @@ function validateUpdatePasswordParams(userInput) {
 		var errorHandler = errorHandlerModule();
 
 		if (!userInput.email) {
-			logger.warn("no email address provided for password update");
+			logger.warn("no email address provided for password update"); console.log(errorHandler.throwInputValidationError("102", "Email is required field"));
 			return reject(errorHandler.throwInputValidationError("102", "Email is required field"));
```

Comment:
Remove console.log , here and other places of code

Reply:
its removed. in my branch.